front-end/src/components/SignUp.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+
+function SignUp(){
+    const [phoneNum, setPhoneNum] = useState("");
+
+    //check to see if the phone number is the correct length and only contains numbers
+    function checkNum(){
+        const reg = /^\d+$/;
+        if(phoneNum.match(reg) && phoneNum.length === 10){
+            return true;
+        }
+        else{
+            return false;
+        }
+    }
+
+    function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+        event.preventDefault();
+        // if a valid phone num, send to back end
+        console.log(checkNum());
```

Comment:
need to delete console.log as it the system should not show the phone number on the log.

Reply:
Alright, will get to it